message/messaging_service.cc

```diff
@@ -1240,6 +1242,28 @@ future<std::vector<row_level_diff_detect_algorithm>> messaging_service::send_rep
     return send_message<future<std::vector<row_level_diff_detect_algorithm>>>(this, messaging_verb::REPAIR_GET_DIFF_ALGORITHMS, std::move(id));
 }
 
+// Wrapper for REPAIR_UPDATE_SYSTEM_TABLE
+void messaging_service::register_repair_update_system_table(std::function<future<repair_update_system_table_response> (const rpc::client_info& cinfo, repair_update_system_table_request)>&& func) {
+    register_handler(this, messaging_verb::REPAIR_UPDATE_SYSTEM_TABLE, std::move(func));
+}
+future<> messaging_service::unregister_repair_update_system_table() {
+    return unregister_handler(messaging_verb::REPAIR_UPDATE_SYSTEM_TABLE);
+}
+future<repair_update_system_table_response> messaging_service::send_repair_update_system_table(msg_addr id, repair_update_system_table_request req) {
+    return send_message<future<repair_update_system_table_response>>(this, messaging_verb::REPAIR_UPDATE_SYSTEM_TABLE, std::move(id), std::move(req));
+}
+
+// Wrapper for REPAIR_FLUSH_HINTS_BATCHLOG
+void messaging_service::register_repair_flush_hints_batchlog(std::function<future<repair_flush_hints_batchlog_response> (const rpc::client_info& cinfo, repair_flush_hints_batchlog_request)>&& func) {
+    register_handler(this, messaging_verb::REPAIR_FLUSH_HINTS_BATCHLOG, std::move(func));
+}
+future<> messaging_service::unregister_repair_flush_hints_batchlog() {
+    return unregister_handler(messaging_verb::REPAIR_FLUSH_HINTS_BATCHLOG);
+}
+future<repair_flush_hints_batchlog_response> messaging_service::send_repair_flush_hints_batchlog(msg_addr id, repair_flush_hints_batchlog_request req) {
+    return send_message<future<repair_flush_hints_batchlog_response>>(this, messaging_verb::REPAIR_FLUSH_HINTS_BATCHLOG, std::move(id), std::move(req));
+}
+
```

Comment:
This boilerplate can be autogenerated (see 0ea79559a669259405265194aa33c1b8f7456ecc)

Reply:
I will try. I hope it works nicely.

Reply:
Don't forget to try. It's pretty nice. It also reduces the need to use structs since the parameters are controlled by IDL.